policyengine_us/parameters/gov/states/wa/tax/income/capital_gains/deductions/charitable/cap.yaml

```diff
@@ -1,12 +1,15 @@
 description: Washington capital gains charitable contribution cap.
 metadata:
   label: Washington capital gains charitable contribution cap
-  name: wa_ltcg_charitable_contributions_cap
   unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.87.080(2) Charitable donation deduction"
+    - title: RCW 82.87.080(2) Charitable donation deduction
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.87.080
+    - title: Capital gains tax | Washington Department of Revenue
```

Comment:
could you add 2022 form as well

Reply:
couldn't find any 2022 form

Reply:
Is there a tax form?

Reply:
seems no 
https://dor.wa.gov/manage-business/my-dor-help/capital-gains-my-dor-help#filereturn